src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -500,16 +500,23 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 	 * @link https://stripe.com/docs/js/elements_object/create_element?type=card#elements_create-options
 	 *
 	 * @since 5.3.0
+	 * @since TBD Pulled out `options` variable to allow filtering using `tec_tickets_commerce_stripe_checkout_localized_data`.
 	 */
 	obj.setupCompactCardElement = () => {
-		// Instantiate the CardElement with a single field combo.
-		obj.cardElement = obj.stripeElements.create( 'card', {
-			style: obj.checkout.cardElementStyle,
-		} );
+		const options = obj.checkout.cardElementOptions;
+
+		// If there are no customized style options being added, use the previously defined default.
+		if( ! options.style ){
+			options.style = obj.checkout.cardElementStyle;
+		}
+
+		// Instantiate the CardElement with the options.
+		obj.cardElement = obj.stripeElements.create( 'card', options );
 		obj.cardElement.mount( obj.selectors.cardElement );
 		obj.cardElement.on( 'change', obj.onCardChange );
 	};
 
+
```

Comment:
Looks good! All expect this line! Kidding :P good to merge!

Reply:
😭  My heart dropped for a second 🤣  Hahahha yay, thanks Dimi!!